src/extension/background-script/actions/nostr/getPublicKey.ts

```diff
@@ -0,0 +1,9 @@
+import { Message } from "~/types";
+
+const getPublicKey = async (message: Message) => {
+  return {
+    data: "4a8c13dd19bff29de83e26ab0b73441d18e39cc3838e323eecc12502c4698beb",
```

Comment:
Hmmm

Reply:
It's WIP, key will be derived from your node's key material.